test/e2e-ansible/e2e_ansible_suite_test.go

```diff
@@ -57,37 +50,12 @@ var _ = BeforeSuite(func(done Done) {
 	By("creating the repository")
 	Expect(tc.Prepare()).To(Succeed())
 
-	By("checking the cluster type")
-	kubectx, err = tc.Kubectl.Command("config", "current-context")
+	By("getting the cluster Kind")
+	tc.Kubectx, err = tc.Kubectl.Command("config", "current-context")
 	Expect(err).NotTo(HaveOccurred())
 
-	By("checking API resources applied on Cluster")
-	output, err := tc.Kubectl.Command("api-resources")
-	Expect(err).NotTo(HaveOccurred())
-	if strings.Contains(output, "servicemonitors") {
-		isPrometheusManagedBySuite = false
-	}
-	if strings.Contains(output, "clusterserviceversions") {
-		isOLMManagedBySuite = false
-	}
-
-	if isPrometheusManagedBySuite {
-		By("installing Prometheus")
-		Expect(tc.InstallPrometheusOperManager()).To(Succeed())
-
-		By("ensuring provisioned Prometheus Manager Service")
-		Eventually(func() error {
-			_, err := tc.Kubectl.Get(
-				false,
-				"Service", "prometheus-operator")
-			return err
-		}, 3*time.Minute, time.Second).Should(Succeed())
-	}
-
-	if isOLMManagedBySuite {
-		By("installing OLM")
-		Expect(tc.InstallOLMVersion(testutils.OlmVersionForTestSuite)).To(Succeed())
-	}
+	By("preparing the prerequisites on cluster")
```

Comment:
wonder if it might be better to stick the By() into InstallPrerequisites() instead of repeating it every time. you already do for the OLM check.

Reply:
The By is part of the test. I mean to describe the step called and the step is the func.
So, IMO would be fine keep as it is.

Reply:
See the other scenarios as well

```
By("turning off interactive prompts for all generation tasks.")
	err = tc.DisableOLMBundleInteractiveMode()
	Expect(err).NotTo(HaveOccurred())
```

so, it would following the current standard.